lib/collection/cookie.js

```diff
@@ -267,11 +270,11 @@ _.extend(Cookie, /** @lends Cookie */ {
         key = split[0].trim();
         value = (split[1]) ? split[1].trim() : true;
 
-        if (_.isString(value) && '"' === value[0]) {
+        if (_.isString(value) && value[0] === QUOT) {
             value = value.slice(1, -1);
         }
 
-        return { key: key, value: value };
+        return {key: key, value: value};
```

Comment:
Yes. I felt icky too and then was wondering that this was present in other places. I agree that we should go to one with spaces.